### PR TITLE
uncomment validation and test notebook breakage

### PR DESCRIPTION
**Pull request by chiayi**

### CODE REVIEW

**Comment by chiayi** [comment]
Forcing the notebook run to fail gave an error in the cloudbuild: 
```
rag frontend test failed
```

**Comment by chiayi** [comment]
Running without forced breakage.

**Comment by chiayi** [comment]
Input the wrong path to rag test.

**Review by imreddy13 — APPROVED** [review]

**Comment by chiayi** [comment]
Hmm it looks like it timed out waiting for jupyter notebook pod to spin up.... going to double to time and retry

File: cloudbuild.yaml [hunk 1]
```diff
@@ -373,11 +373,11 @@ steps:
           exit 1
         fi
 
-        # TODO: re-add this check once generating embeddings in CI is stable
-        # if [[ $(cat /workspace/rag_prompt_result.txt) != "pass" ]]; then
-        #   echo "rag frontend test failed"
-        #   exit 1
-        # fi
+        TODO: re-add this check once generating embeddings in CI is stable
```
Comment (andrewsykim) [inline cloudbuild.yaml:376]: This TODO was previously a comment and nowshould be removed entirely

Comment (andrewsykim) [reply cloudbuild.yaml:376]: This is breaking CI too:
```
Already have image (with digest): gcr.io/gke-ai-eco-dev/terraform
bash: line 46: TODO:: command not found
```

**Pull request merged**